experiments/yarn/src/test/resources/gear.conf

```diff
@@ -0,0 +1,165 @@
+#########################################
```

Comment:
@TomaszT 

"gear.conf" should not be included here.

If you want to create some default configuration file, you can use file name "reference.conf",
and only contains settings which is interesting to gearpump-yarn.